frameworks/kafka/src/main/java/com/mesosphere/sdk/kafka/api/TopicController.java

```diff
@@ -0,0 +1,171 @@
+package com.mesosphere.sdk.kafka.api;
+
+import com.mesosphere.sdk.kafka.cmd.CmdExecutor;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Topic Controller executing commands through command executor.
+ * Kafka package should also be deployed with framework to enable those kafka commands.
+ */
+
+@Path("/v1/topics")
+public class TopicController {
```

Comment:
I think `Resource` is referring to what `Controller` is referring to (e.g. a grouping of endpoint handler logic) so `TopicResource` or `TopicsResource` seems most consistent with the other classes in this area.